packages/smart-contracts/src/contracts/ERC20TransferrableReceivable.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/utils/Counters.sol';
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol';
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol';
+
+contract ERC20TransferrableReceivable is ERC721, ERC721Enumerable, ERC721URIStorage {
```

Comment:
Please fix all the places where "Transferable" is mispelled with 2 `r`s. This comment applies to the whole PR. https://grammarhow.com/transferrable-or-transferable/

I just learned that it's spelled "Transferable" with 1 `r`. My bad if I mispelled it in the past :sweat-smile: